File: src/3375.minimum-operations-to-make-array-values-equal-to-k.3621/solution.java
class Solution {
    public int minOperations(int[] nums, int k) {
        int count = (int)Arrays.stream(nums).distinct().count();
        
        int min = nums[0];
        for(int n:nums){
            min = Math.min(min,n);
        }
        if(k>min){
            return -1;
        }
        return count-( k<min?0:1);
    }
}

File: src/3375.minimum-operations-to-make-array-values-equal-to-k.3621/solution.ts
function minOperations (nums: number[], k: number): number {
    nums = [...new Set(nums), ];
    nums.sort((a, b) => a - b);
    if (nums[0] < k) {
        return -1;
    }
    return nums.length - (nums[0] === k ? 1 : 0);
}
